template/test-report.vuln-card.hbs

```diff
@@ -70,80 +70,86 @@
 
         <hr/>
 
-        {{#if_eq @root.project.type "npm"}}
-            <h3 class="card__section__title">Detailed paths and remediation</h3>
-        {{else}}
-            <h3 class="card__section__title">Detailed paths</h3>
-        {{/if_eq}}
-
-        <ul class="card__meta__paths">
-            {{#each list}}
-                {{#isDoubleArray from}}
-                    {{#each .}}
-                    <li class="list-paths__item">
+        {{#unless showSummaryOnly}}
```

Comment:
could you add a screenshot with & without `-s` to the PR description please to help visualise

Reply:
sure! description updated